javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/PropagatorsInitializer.java

```diff
@@ -50,6 +53,7 @@ public static void initializePropagators(List<String> propagators) {
       OpenTelemetry.setGlobalPropagators(
           DefaultContextPropagators.builder()
               .addTextMapPropagator(HttpTraceContext.getInstance())
+              .addTextMapPropagator(W3CBaggagePropagator.getInstance())
```

Comment:
@carlosalberto 
`W3CBaggagePropagator` should be first one or last one? 
it doesn't matter for inject but for extract we return context from last propagator.

Reply:
It does't matter for the default configuration case 😄

Reply:
@carlosalberto does it matter the order we add `W3CBaggagePropagator` for the non-default configuration case?

Reply:
@trask So `Baggage` propagators work just like the Trace ones (B3, TraceContext, etc), regarding the fact that, upon successful extraction, any previous value will be overridden (i.e. that last one wins).

Currently only the Jaeger propagator does `Baggage` extraction besides `W3CBaggage`, so the order _may_ matter in such case, e.g. if the user explicitly specifies `OTEL_PROPAGATORS=jaeger,baggage`, then `W3CBaggage` will have higher priority, given the case that `Baggage` was propagated suing **both** `Jaeger` and `W3CBaggage` formats.

Then again, when users specify this, they should know what they are doing, IMHO 😉 (although maybe we should add a note in OTel Java regarding this, etc).

Reply:
@carlosalberto do u know what we should do if the user specifies:

`OTEL_PROPAGATORS=jaeger,baggage,tracecontext`

Should we add both TraceMultiPropagator(jaeger,tracecontext) and W3BaggagePropagator? And if so, do u know whether we should add TraceMultiPropagator first or last?

Reply:
After thinking a little bit more about this, I think we should leave `JaegerPropagator` out of the `TraceMultiPropagator` given that it **not only** extracts `SpanContext`. For example, `TraceMultiPropagator` with "tracecontext,jaeger" will stop if trace-context succeeded, without giving `JaegerPropagator` a chance to extract any existing `Baggage` ;)

It would mean, given your example, that you would end up with Jaeger + Baggage + TraceContext (no TraceMultiPropagator needed here).

Other than that, the order doesn't matter much - although `W3CBaggagePropagator` should probably come _after_ `JaegerPropagator`, as it can have more complex/complete values that Jaeger baggage lacks, e.g. metadata.

(We could split `JaegerPropagator` into a trace AND a baggage propagators but that slightly seems like an overkill)

Reply:
@malafeev ^ ;)

Reply:
so `JaegerPropagator` always should be added via `addTextMapPropagator`?
And next example is valid?:
```
DefaultContextPropagators.builder()
    .addTextMapPropagator(JaegerPropagator.getInstance())
    .addTextMapPropagator(W3CBaggagePropagator.getInstance()) 
    .addTextMapPropagator(TraceMultiPropagator.builder()
        .addPropagator(HttpTraceContext.getInstance())
        .addPropagator(AwsXRayPropagator.getInstance()).build())
    ...
```
?

Reply:
@malafeev  Yes ;) - and put a comment in the code on why we are doing it (i.e. Jaeger handles BOTH tracing and baggage, etc).

Reply:
done

Reply:
@malafeev I think both users and developers will have a hard time understanding how `otel.propagators` works now :) Please document the final result both in `PropagatorsInitializer.java` and in README. The latter, for example, currently does very poor job explaining the difference between trace context propagators and baggage ones.

Reply:
@iNikem I added javadoc, is it clear enough?

Reply:
Sorry, but no :) I will add comments to javadoc. And we still have update README